data/Makefile

```
@@ -648,6 +648,18 @@ minsize=2000
 %.histo: %.hist
 	sed -n 's/^f//p' $< | tr '\t' ' ' >$@
 
+# Count k-mers using ntCard.
+%_k$k.histogram.txt: %.fq.gz
+	$(time) ntcard -t $t -k $k -o $@ $<
+
+# Generate repetitive k-mers using ntHits.
+%_k$k.rep: %.fq.gz %_k$k.histogram.txt
+	$(eval mode := $(shell $(python) $(bin)/physlr find-ntcard-mode $*_k$k.histogram.txt ))
+	>&2 echo "The mode k-mer coverage is $(mode)."
```

We'll want `@echo` here so that it doesn't print out that line before it executes it. Do we need the `>&2`?

---

For the second part I'm not sure. Im ok with piping it to stdout by default, but I figured some people might only use `2> x.log`

---

```suggestion
	@>&2 echo "The mode k-mer coverage is $(mode)."
```
if you want to keep the stderr pipe